hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/netty/DataBufferMarshaller.java

```diff
@@ -41,7 +41,9 @@ final class DataBufferMarshaller implements MethodDescriptor.Marshaller<Buffered
      * have control over the thread pool used by the underlying gRPC server.
      */
     private static final ThreadLocal<BufferedData> BUFFER_THREAD_LOCAL =
-            ThreadLocal.withInitial(() -> BufferedData.allocate(TOO_BIG_MESSAGE_SIZE));
+            ThreadLocal.withInitial(() -> BufferedData.allocate(
+                    TOO_BIG_MESSAGE_SIZE)); // TODO: should we address this "ThreadLocal" variables should be cleaned up
+    // when no longer used i.e. call remove() on BUFFER_THREAD_LOCAL somewhere?
```

Comment:
I'm only speaking for me, but if the answer isn't known, it seems better to leave the sonar warning instead of adding a `TODO` in the code

Reply:
After checking with @rbair23 those two `ThreadLocal` related warnings in `hedera-app` will be suppressed as those are false positives reported by Sonar.